specification/components/examples/ResponseRequestForDocumentsAccess.yaml

```diff
@@ -1,9 +1,10 @@
 summary: |
-  Request for documents access for historical permissions that is pending.
+  A pending response from a request where the patient asked for detailed access
+  to the historical medical record.
 value:
   permissionType: medicalRecord
   subType: historical
-  accessLevel: documents
+  accessLevel: detailed
   status: pending
   message: |
     Request accepted to update access historical effective date to documents.
```

Comment:
does this message need to be updated? it sounds a bit like the requested change has been accepted

Reply:
Yeah, this message is a bit confused!